Moderation-Policy.md

```diff
@@ -0,0 +1,129 @@
+## Moderation policy
+
+If you are not a member of the Node.js GitHub Organization and wish to submit a moderation request, please see [Requesting Moderation](#requesting-moderation)
+
+* [Applicability](#applicability)
+* [Terms](#terms)
+* [Grounds for Moderation](#grounds-for-moderation)
+* [Requesting Moderation](#requesting-moderation)
+* [Consideration of Intent](#consideration-of-intent)
+* [Policy](#policy)
+ * [Collaborator Posts](#collaborator-posts)
+ * [Non-Collaborator Posts](#non-collaborator-posts)
+* [Escalation to the TSC](#escalation-to-the-tsc)
+* [Privacy of the nodejs/moderation Repository](#privacy-of-the-nodejsmoderation-repository)
+* [TSC Delegation of Authority to a "Moderation Working Group"](tsc-delegation-of-authority-to-a-moderation-working-group)
+* [Modifications to this Policy](#modifications-to-this-policy)
+
+### Applicability
+
+By default, this policy applies to all repositories under the Node.js GitHub Organization and all Node.js Working Groups.
+
+Individual Working Groups and Top Level Projects chartered by the TSC may adopt an alternative Moderation Policy for any repository under their stewardship so long as:
+* The Moderation Policy is openly documented as part of the Working Group or Project Charter and;
+* Includes provisions for clearly and openly documenting Moderation actions taken.
+
+If a particular repository can be considered to fall under the stewardship of multiple Working Groups (or Top Level Projects) that have adopted different Moderation Policies, they can choose to either:
+* Decide between themselves which Moderation Policy will be in effect for the repository in question, or
+* Ask the TSC to determine which Moderation Policy should apply.
+
+Any alternative Moderation Policy used for a given repository must be included in the root directory of the repository using the `Moderation-Policy.md` filename. If a repository does not contain a `Moderation-Policy.md` file, then this default policy applies.
+
+### Terms
+
+* "Collaborator" refers to any individual with configured access permissions to any Node.js GitHub organization repository. See [GitHub's access permissions documentation](https://help.github.com/articles/what-are-the-different-access-permissions/) for more information.
+* "TSC" refers to the [Node.js Technical Steering Committee](https://github.com/nodejs/node#tsc-technical-steering-committee).
+* "Post" refers to the content and titles of any issue, pull request, comment or wiki page.
+* "Moderate" refers to the act of modifying the content and title of, or deleting, any Post.
```

Comment:
Personally I'd prefer that no one make any edits in someone else's posts
unless they're specifically for moderation, good faith or otherwise.
On Nov 23, 2015 11:00 PM, "Brendan Ashworth" <notifications@github.com>
wrote:

> In Moderation-Policy.md
> <https://github.com/nodejs/TSC/pull/16#discussion_r45702032>:
>
> > +Individual Working Groups and Top Level Projects chartered by the TSC may adopt an alternative Moderation Policy for any repository under their stewardship so long as:
> > +* The Moderation Policy is openly documented as part of the Working Group or Project Charter and;
> > +* Includes provisions for clearly and openly documenting Moderation actions taken.
> > +
> > +If a particular repository can be considered to fall under the stewardship of multiple Working Groups (or Top Level Projects) that have adopted different Moderation Policies, they can choose to either:
> > +* Decide between themselves which Moderation Policy will be in effect for the repository in question, or
> > +* Ask the TSC to determine which Moderation Policy should apply.
> > +
> > +Any alternative Moderation Policy used for a given repository must be included in the root directory of the repository using the `Moderation-Policy.md` filename. If a repository does not contain a `Moderation-Policy.md` file, then this default policy applies.
> > +
> > +### Terms
> > +
> > +* "Collaborator" refers to any individual with configured access permissions to any Node.js GitHub organization repository. See [GitHub's access permissions documentation](https://help.github.com/articles/what-are-the-different-access-permissions/) for more information.
> > +* "TSC" refers to the [Node.js Technical Steering Committee](https://github.com/nodejs/node#tsc-technical-steering-committee).
> > +* "Post" refers to the content and titles of any issue, pull request, comment or wiki page.
> > +* "Moderate" refers to the act of modifying the content and title of, or deleting, any Post.
>
> Is this intended to also include smaller, good-faith edits like typos? Or
> would those be excluded?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/TSC/pull/16/files#r45702032>.
>